base/docs/helpdb.jl

```diff
@@ -16340,3 +16340,14 @@ If ``pkg`` is installed, return the installed version number, otherwise return `
 """
 Pkg.installed
 
+doc"""
+```rst
+::
+           randjump(r::MersenneTwister, jumps, [jumppoly]) -> Vector{MersenneTwister}
+
+Create an array of the size ``jumps`` of initialized ``MersenneTwister`` RNG objects where the first RNG object in the array is initialized with ``seed`` and following ``MersenneTwister`` RNGs in the array initialized such that a state of the RNG object in the array would be moved forward (without generating numbers) from a previous RNG object array element on a particular number of steps encoded by the jump polynomial ``jumppoly``.
+
+Default jump polynomial moves forward ``MersenneTwister`` RNG state on 10^20 steps.
+```
+"""
+randjump
```

Comment:
I think you need to add a signature somewhere in the RST docs for `doc/genstdlib.jl` to populate the docstring correctly